content/management/changelog/index.mdx

```diff
@@ -49,14 +49,25 @@ Since the migration is still ongoing, different parts will be successively moved
 
 ### Webhooks
 
-- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method had the `webhook_id` parameter renamed to `id`.
-- The **customer_created** webhook was renamed to [**incoming_customer**](/management/configuration-api/v3.3/#incoming_customer).
-- The [**customer_session_fields_updated**](/management/configuration-api/v3.3/#customer_session_fields_updated) webhook was added.
-- The [**List Webhook Names**](/management/configuration-api/v3.3/#list-webhook-names) method was added.
+- The [**Unregister Webhook**](/management/configuration-api/v3.3/#unregister-webhook) method:
+  - had the `webhook_id` parameter renamed to `id`.
+  - had the required scope changed to `webhooks.configuration:rw`.
+  - has a new mandatory parameter `owner_client_id`.
+- The **List Registered Webhooks** method:
+  - was renamed to [**List Webhooks**](/management/configuration-api/v3.3/#list-webhooks).
+  - had the required scope changed to `webhooks.configuration:rw`.
+  - has a new mandatory parameter `owner_client_id`.
+- The [**Register Webhook**](/management/configuration-api/v3.3/#register-webhook) method:
```

Comment:
I would also describe the main changes in these 3 methods from the user perspective:
* allow to register a webhook definition only (template), enabling webhooks for a license is another action (can be automatically enabled or manually)
* allow managing separate webhooks only for v3.3+ - you can; 't list old webhooks with new methods - it's very important!! And the developer must migrate old webhooks to the new ones.